hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieInputFormatUtils.java

```diff
@@ -82,6 +82,13 @@
   public static final int HOODIE_PARTITION_PATH_COL_POS = 3;
   public static final String HOODIE_READ_COLUMNS_PROP = "hoodie.read.columns.set";
 
+  public static final String HOODIE_USE_META_FIELDS = "hoodie.use.meta.fields";
```

Comment:
rational to use a different naming compared to what we use in table props. This is just a query side variable used just in realtime path. "UseMetaFields" makes more sense from a query standpoint than "hoodiePopulateMetaFields".

Reply:
but its a table level property and the query has the understandin that its about data is stored, not just about how its written. I would advise against having a different name for the same thing. it leads to more harm than good over the long run

Reply:
I am not sure If I fully go with that just in this specific context. In general, I agree having the same property throughout makes sense. but here, somehow I feel its not very apparent. 
 
populateMetaFields is something clearly on the write path(as it conveys whether to populate meta fields or not). I would expect something like "hasMetaFields" or something for read path. 
Or we could rename the entire property to "enableMetaFields" or something so that both write and read side makes sense. 
Anyways, w/ latest fix, we may not need these variables only. will remove them. just wanted to convey my opinion.